baseProject/src/main/webapp/pages/template/~bottom.jsp

```diff
@@ -6,6 +6,57 @@
 <script src="/resources/js/prod/${param.js}.js"></script>
 <script>
 
+	function onSignIn(googleUser) {
+		var profile = googleUser.getBasicProfile();
+
+		var gUser = {
+			id_token: googleUser.getAuthResponse().id_token,
+			firstname: profile.getGivenName(),
+			lastname: profile.getFamilyName(),
+			email: profile.getEmail(),
+			picture: profile.getImageUrl()
+		};
+
+
+		// Send stuff to controller just like the example below. So, specify
+		// the url, e.g '/signup/doesEmailExist' and then specify an obj with
+		// key:value pairs ({key1: value, key2: value, ...}) where the keys
+		// correspond to the properties of the form that is handled in a
+		// controller method. I.e. if the signupForm has a property firstname, the
+		// key to be passed to the controller method has to be the same. Of course,
+		// types of the values should match too, i guess.
+
+		// EXAMPLE
+		// check if email exists - and handle according response (bool)
+		$.post('/signup/doesEmailExist', {email: gUser.email})
+			.done(function (existingEmail) {
+				if (existingEmail) {
+					// do somn, i.e. send stuff to another controller the same way
+
+
+				} else {
+					// do somn else
+
+
+				}
+			});
+
+	}
+
```

Comment:
NB:
- The url corresponds to the one used in RequestMapping.
- In this specific example the returned response data 'existingEmail' is a boolean. It does not have to be a bool, it could be a string or a banana depending on what the method returns... of course.
- You might want to add other 'events' to properly handle the outcome of the post: .fail(func).always(func)